.travis.yml

```diff
@@ -1,7 +1,4 @@
-sudo: false
 before_install: gem install bundler
 rvm:
-  - '2.3.8'
-  - '2.5.3'
   - '2.6.0'
-
+  - '2.7.2'
```

Comment:
Why add this version?

Reply:
Why not? It's the next release, so we should also consider supporting it

Ruby 2.7
status: normal maintenance
release date: 2019-12-25

Ruby 2.6
status: normal maintenance
release date: 2018-12-25

Ruby 2.5
status: security maintenance
release date: 2017-12-25
EOL date: 2021-03-31